docs/keycodes.md

```diff
@@ -381,45 +381,51 @@ See also: [Mouse Keys](feature_mouse_keys.md)
-|`LGUI_T(kc)` |`LCMD_T(kc)`, `LWIN_T(kc)`, `GUI_T(kc)`, `CMD_T(kc)`, `WIN_T(kc)`|Left GUI when held, `kc` when tapped                   |
-|`RCTL_T(kc)` |                                                                 |Right Control when held, `kc` when tapped              |
-|`RSFT_T(kc)` |                                                                 |Right Shift when held, `kc` when tapped                |
-|`RALT_T(kc)` |`ROPT_T(kc)`, `ALGR_T(kc)`                                       |Right Alt when held, `kc` when tapped                  |
-|`RGUI_T(kc)` |`RCMD_T(kc)`, `RWIN_T(kc)`                                       |Right GUI when held, `kc` when tapped                  |
-|`SGUI_T(kc)` |`SCMD_T(kc)`, `SWIN_T(kc)`                                       |Left Shift and GUI when held, `kc` when tapped         |
-|`LCA_T(kc)`  |                                                                 |Left Control and Alt when held, `kc` when tapped       |
-|`LCAG_T(kc)` |                                                                 |Left Control, Alt and GUI when held, `kc` when tapped  |
-|`RCAG_T(kc)` |                                                                 |Right Control, Alt and GUI when held, `kc` when tapped |
-|`C_S_T(kc)`  |                                                                 |Left Control and Shift when held, `kc` when tapped     |
-|`MEH_T(kc)`  |                                                                 |Left Control, Shift and Alt when held, `kc` when tapped|
+|Key          |Aliases                                                          |Description                                                   |
+|-------------|-----------------------------------------------------------------|--------------------------------------------------------------|
+|`MT(mod, kc)`|                                                                 |`mod` when held, `kc` when tapped                             |
+|`LCTL_T(kc)` |`CTL_T(kc)`                                                      |Left Control when held, `kc` when tapped                      |
+|`LSFT_T(kc)` |`SFT_T(kc)`                                                      |Left Shift when held, `kc` when tapped                        |
+|`LALT_T(kc)` |`LOPT_T(kc)`, `ALT_T(kc)`, `OPT_T(kc)`                           |Left Alt when held, `kc` when tapped                          |
+|`LGUI_T(kc)` |`LCMD_T(kc)`, `LWIN_T(kc)`, `GUI_T(kc)`, `CMD_T(kc)`, `WIN_T(kc)`|Left GUI when held, `kc` when tapped                          |
+|`RCTL_T(kc)` |                                                                 |Right Control when held, `kc` when tapped                     |
+|`RSFT_T(kc)` |                                                                 |Right Shift when held, `kc` when tapped                       |
+|`RALT_T(kc)` |`ROPT_T(kc)`, `ALGR_T(kc)`                                       |Right Alt (Alt Gr) when held, `kc` when tapped                |
```

Comment:
```suggestion
|`RALT_T(kc)` |`ROPT_T(kc)`, `ALGR_T(kc)`                                       |Right Alt (AltGr) when held, `kc` when tapped                 |
```

Reply:
This is in documentation.  **Alt Gr** is typically listed as two "words," not one, with a space in between.

Reply:
What documentation?

https://en.wikipedia.org/wiki/AltGr_key

Reply:
I mean that these changes above are in QMK's documentation, so they are user-visible.  It's interesting that Wikipedia lists it as "AltGr" even though actual keyboards tend far more often to put a space in between the two.

I guess I concede the point.

Reply:
I couldn't find the sequence "Alt Gr" anywhere in the docs.

Reply:
Again, I'm noting that the code changes that we are currently commenting on happen to be in files that will be provided as QMK's user-facing documentation.

I've already accepted your suggestions, but for reference, keyboards that have an Alt Gr key tend to show the space between "Alt" and "Gr" in the legend.  [Example German keyboard](https://en.wikipedia.org/wiki/German_keyboard_layout#/media/File:German-T2-Keyboard-Prototype-May-2012.jpg).